Fix composer warning about uppercase in require block

hnnweb opened the pull request:
Fix composer warning about uppercase in require block

```
$ composer install
Deprecation warning: require.rmccue/Requests is invalid, it should not contain uppercase characters. Please use rmccue/requests instead. Make sure you fix this as Composer 2.0 will error.
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
```

ozh commented:
Sure. I'll let @LeoColomb say if there is potential impact to this, I'm not even sure what we really need this composer file for (should the composer lock file be updated as well? other composer include files?)

hnnweb commented:
Updated the lock file. Ran `composer update nothing`

hnnweb commented:
It actually wrote 
```
	modified:   includes/vendor/autoload.php
	modified:   includes/vendor/composer/ClassLoader.php
	modified:   includes/vendor/composer/LICENSE
	modified:   includes/vendor/composer/autoload_real.php
	modified:   includes/vendor/composer/installed.json
```

this aswell. Want them too?
